internal/util/vault.go

```diff
@@ -58,6 +58,10 @@ Example JSON structure in the KMS config is,
 		"vaultNamespace": "",
 		"vaultPassphraseRoot": "/v1/secret",
 		"vaultPassphrasePath": "",
+		// `vaultToken` is not very secure or not recommended to use in production,
```

Comment:
I assume we are not adding this to `docs/deploy-rbd.md` as this is not a user facing configuration that we want to provide/encourage, right?

Reply:
Yes. we are not mentioning it in the deploy-rbd.md .